### PR TITLE
Now you can login and sign up with Google :)

### PR DESCRIPTION
**Pull request by amandasavluchinske**
Files to review: pokebattle/settings/base.py, pokebattle/urls.py, templates/accounts/login.html, templates/accounts/login.html

### CODE REVIEW

File: assets/sass/_accounts.scss [hunk 1]
```diff
@@ -21,4 +21,50 @@
...
+    padding: 0 15px 0 46px;
+    border: none;
+    text-align: left;
+    line-height: 34px;
+    white-space: nowrap;
+    border-radius: 0.2em;
+    font-size: 16px;
+    color: #FFF;
+    a {
+        color: white;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: The color variable shouldn't be used directly here. The idea of sass variables is being able to modify them by accessing only the _variables.scss file. Ideally there would be a variable on the file that would be something like:

```sass
// _variables.scss
$account-loggin-btn-color: $color-white;
```

File: assets/sass/_accounts.scss [hunk 2]
```diff
@@ -21,4 +21,50 @@
...
+    position: relative;
+    /* width: 13em;  - apply for fixed size */
+    margin: 0.2em;
+    padding: 0 15px 0 46px;
+    border: none;
+    text-align: left;
+    line-height: 34px;
+    white-space: nowrap;
+    border-radius: 0.2em;
+    font-size: 16px;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: Add this to a variable. Don't use it directly here. General things should always be in variables

File: assets/sass/_accounts.scss [hunk 3]
```diff
@@ -21,4 +21,50 @@
...
+    color: #FFF;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: Same here

File: assets/sass/_accounts.scss [hunk 4]
```diff
@@ -21,4 +21,50 @@
...
+    box-sizing: border-box;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 34px;
+    height: 100%;
+  }
+  .loginBtn:focus {
+    outline: none;
+    color: white;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: Here

File: assets/sass/_accounts.scss [hunk 5]
```diff
@@ -21,4 +21,50 @@
...
+  }
+  .loginBtn:active {
+    box-shadow: inset 0 0 0 32px rgba(0,0,0,0.1);
+  }
+  /* Google */
+.loginBtn--google {
+    font-family: "Roboto", Roboto, arial, sans-serif;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: This should be on a variable too, preferably on a global file (`global.scss` or `custom_bootstrap.scss`)

File: assets/sass/_accounts.scss [hunk 6]
```diff
@@ -21,4 +21,50 @@
...
+    background: #DD4B39;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: Same

File: assets/sass/_accounts.scss [hunk 7]
```diff
@@ -21,4 +21,50 @@
...
+  }
+  .loginBtn--google:before {
+    border-right: #BB3F30 1px solid;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: Ping :ping_pong:

File: assets/sass/_accounts.scss [hunk 8]
```diff
@@ -21,4 +21,50 @@
...
+    background: url('https://s3-us-west-2.amazonaws.com/s.cdpn.io/14082/icon_google.png') 6px 6px no-repeat;
+  }
+  .loginBtn--google:hover,
+  .loginBtn--google:focus {
+    background: #E74B37;
```
Comment (rsarai) [inline assets/sass/_accounts.scss]: Pong :ping_pong:

File: assets/sass/_accounts.scss (hunk 2 above)
Comment (amandasavluchinske) [reply assets/sass/_accounts.scss]: What should I add to the variable? I didn't understand.

**Comment by amandasavluchinske** [comment]
should be ok now @rsarai

**Pull request merged**